docs/GETTINGSTARTED.md

```diff
@@ -88,6 +88,8 @@ Where:
 
 The `token` mentioned above must be for a service account which is able to list and get ingress resources in the Kube cluster. We can create a service account specifically for Yggdrasil with the below ClusterRole and matching ClusterRoleBinding.
 
+`Note:` Please see the [Kubernetes docs](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#service-account-tokens) for greater detail on creating and using service account tokens.
+
```

Comment:
I might be worth explicitly calling out that once you've created the service account you need to get the token from the secret to give to yggdrasil